.github/workflows/release.yaml

```diff
@@ -2,7 +2,7 @@ name: Release Flow
 
 on:
   release:
-    types: [published]
+    types: [prereleased, released]
```

Comment:
Can you explain a bit this change? Why are we removing published?

Reply:
to work with release and pre  release individualy